tests/test_stt.py

```diff
@@ -23,51 +23,23 @@
 
 sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
 
-from neon_stt_plugin_polyglot import PolyglotSTT
+from neon_stt_plugin_coqui import CoquiSTT
 from ovos_utils.log import LOG
 import neon_utils.parse_utils
 import unittest
 from jiwer import cer
-import re
 
 ROOT_DIR = os.path.dirname(os.path.realpath(__file__))
 TEST_PATH_EN = os.path.join(ROOT_DIR, "test_audio/en")
 TEST_PATH_FR = os.path.join(ROOT_DIR, "test_audio/fr")
 TEST_PATH_ES = os.path.join(ROOT_DIR, "test_audio/es")
 TEST_PATH_PL = os.path.join(ROOT_DIR, "test_audio/pl")
 
-def transliteration(transcription, text, lang):
-    transliterated = []
-    translit_dict = {}
-    if lang == 'pl':
-        translit_dict = {'a': ['ą'], 'c': ['ć'], 'e': ['ę'], 'n': ['ń'], 'o': ['ó'], 's': ['ś'], 'z': ['ź', 'ż'], 'l': ['ł']}
-    if lang == 'fr':
-        translit_dict = {'c': ['ç'], 'e': ['é', 'ê', 'è', 'ë'], 'a': ['â', 'à'], 'i': ['î', 'ì', 'ï'],
-                         'o': ['ô', 'ò'], 'u': ['û', 'ù', 'ü']}
-    if lang == 'es':
-        translit_dict = {'a': ['á'], 'i': ['í'], 'e': ['é'], 'n': ['ñ'], 'o': ['ó'], 'u': ['ú', 'ü']}
-    if lang == 'de':
-        translit_dict = {'a': ['ä'], 's': ['ß'], 'o': ['ö'], 'u': ['ü']}
-    transcription = re.sub("`|'|-", "", transcription)
-    text = re.sub("`|'|-", "", text)
-    if len(transcription.strip()) == len(text.strip()):
-        for ind, letter in enumerate(transcription):
-            if letter in translit_dict.keys():
-                if letter != text[ind]:
-                    for l in translit_dict[letter]:
-                        if l == text[ind]:
-                                transliterated.append(l)
-                else:
-                        transliterated.append(letter)
-            else:
-                    transliterated.append(letter)
-        translit_str = ''.join(transliterated)
-        return translit_str
-    else:
-        return text
+COQUI_CREDENTIALS = '/home/mariia/neon-stt-plugin-polyglot/tests/coqui_models.jsonl'
```

Comment:
This can be made into a relative path so it works everywhere, though I'm not sure its used anywhere anyways?

```python
COQUI_CREDENTIALS=os.path.join(os.path.dirname(__file__), "coqui_models.jsonl")
```

Reply:
done